bundles/org.openhab.binding.opensprinkler/README.md

```diff
@@ -36,42 +27,54 @@ Bridge opensprinkler:http:http [hostname="127.0.0.1", port=80, password="opendoo
-|                    |             |    | switched on. It is advised to add persistence for items  |
-|                    |             |    | linked to this channel, the binding does not persist     |
-|                    |             |    | values of it.                                            |
+| nextDuration       | Number:Time | RW | The amount of time that will be used to keep the station |
+|                    |             |    | open when next manually switched on. If not set, this    |
+|                    |             |    | value will default to 18 hours which is the maximum time | 
+|                    |             |    | supported.                                               |
 | queued             | Switch      | RW | Indicates that the station is queued to be turned on.    |
 |                    |             |    | The channel cannot be turned on, only turning it off is  |
 |                    |             |    | supported (which removes the station from the queue).    |
+| ignoreRain         | Switch      | RW | This channel makes the station ignore the rain delay.    |
 
 When using the `nextDuration` channel, it is advised to setup persistence (e.g. MapDB) in order to persist the value through restarts.
 
-The following is supported by the `device` thing, but only when connected using the http interface.
+The following channels are supported by the `device` thing.
+NOTE: Some channels will only show up if the hardware has the required sensor and is setup correctly.
 
 | Channel Type ID | Item Type              |    | Description                                                                        |
 |-----------------|------------------------|----|------------------------------------------------------------------------------------|
 | rainsensor      | Switch                 | RO | This channel indicates whether rain is detected by the device or not.              |
-| currentDraw     | Number:ElectricCurrent | RO | Shows the current draw of the device. If the device does not have sensors          |
-|                 |                        |    | for this metric, the channel will not be available.                                |
+| sensor2         | Switch                 | RO | This channel is for the second sensor (if your hardware supports it).              |
+| currentDraw     | Number:ElectricCurrent | RO | Shows the current draw of the device.                                              |
 | waterlevel      | Number:Dimensionless   | RO | This channel shows the current water level in percent (0-250%). The water level is |
 |                 |                        |    | calculated based on the weather and influences the duration of the water programs. |
-
-## Example
-
-demo.Things:
+| signalStrength  | Number:Power           | RO | Shows the reported RSSI value in dB to indicate how strong the WiFi Signal is.     |
```

Comment:
Just a curious question: For what would such an information be used?

Reply:
I agree it may not be the most useful, but added it because other home automation platforms offer this ability and also since sprinkler controllers are usually outside they may suffer from bad wifi connectivity and this may help diagnose issues for some people. I have made the channel advanced.